src/neptune/handler.py

```diff
@@ -449,19 +448,17 @@ def extend(
 
     @check_protected_paths
     def add(self, values: Union[str, Iterable[str]], *, wait: bool = False) -> None:
-        """Adds the provided tag or tags to the run's tags.
+        """Adds the provided tags to the run.
 
         Args:
             values (str or collection of str): Tag or tags to be added.
                 .. note::
-                    If you want you can use emojis in your tags eg. "Exploration 🧪"
-            wait (bool, optional): If `True` the client will wait to send all tracked metadata to the server first.
+                    You can use emojis in your tags eg. "Exploration 🧪"
```

Comment:
I typically avoid abbreviations in documentation.

```suggestion
                    You can use emojis in your tags. For example, "Exploration 🧪"
```